crates/examples/infra/mod.rs

```diff
@@ -190,6 +193,15 @@ pub fn read_orchestrator_init_config<TYPES: NodeType>() -> (
                 .help("Sets the number of fixed leader for gpu vid, only be used when leaders running on gpu")
                 .required(false),
         )
+        .arg(
+            Arg::new("builder")
+                .short('b')
+                .long("builder")
+                .value_name("BUILDER_TYPE")
+                .value_parser(value_parser!(BuilderType))
+                .help("Sets type of builder. `simple` or `random` to run corresponding integrated builder, `external` to use the one specified by `[config.builder_url]` in config")
+                .required(false),
+        )
```

Comment:
Can we perhaps add a default value (if there isn't one)?

Reply:
There is, clap respects `Default` impl for `BuilderType` (which is `Simple`)